src/loadRc.js

```diff
@@ -32,12 +32,18 @@ module.exports = function loadRc(
   }
 
   function parseExtensionlessRcFile(content: ?string): ?cosmiconfig$Result {
-    if (!content) return null;
-    const pasedConfig = options.rcStrictJson
+    if (typeof content !== 'string') return null;
+    if (content === '')
+      return {
+        config: undefined,
+        filepath,
+      };
```

Comment:
Looks like Prettier probably made this uglier than you intended. Could you add some `{}` around the block?

Reply:
Ah sorry about that.  Will do